libcloud/compute/base.py

```diff
@@ -822,7 +822,7 @@ class NodeDriver(BaseDriver):
     name = None  # type: str
     api_name = None  # type: str
     website = None  # type: str
-    type = None  # type: Provider
+    type = None  # type: Union[Provider,str]
```

Comment:
NOTE: Ideally this would always be ``Provider`` enum, but a lot of the existing code uses ``str`` type so we still need to support it for backward compatibility reasons.